maven_install_2.13.json

```diff
@@ -1,6 +1,6 @@
 {
     "dependency_tree": {
-        "__AUTOGENERATED_FILE_DO_NOT_MODIFY_THIS_FILE_MANUALLY": -367049757,
+        "__AUTOGENERATED_FILE_DO_NOT_MODIFY_THIS_FILE_MANUALLY": -536058548,
```

Comment:
make sure you have rerun the pin command to get the latest unique gen'd value here

```bazel run @unpinned_maven//:pin```

Reply:
i did - it wouldn't compile otherwise